client/app/account/billing/autoRenew/billing-autoRenew.controller.js

```diff
@@ -268,6 +268,9 @@ angular.module("Billing.controllers").controller("Billing.controllers.AutoRenew"
             }
         };
 
+        $scope.buildSMSCreditBuyingURL = (service) => `https://www.ovhtelecom.fr/manager/index.html#/sms/${service.serviceId}/order`;
```

Comment:
Whole refactoring of the page coming soon

Reply:
Even if we will refactoring the page, is there any reason to not use this constant instead: https://github.com/ovh-ux/ovh-manager-dedicated/blob/master/constants.config.js#L349

Reply:
Excellent point, I actually looked for something similar but couldn't find it. I'll do the change